### PR TITLE
app-text/po4a: version bump to 0.47

### PR DESCRIPTION
**Pull request by Fat-Zer**
Gentoo-Bug: [550210](https://bugs.gentoo.org/show_bug.cgi?id=550210)

Note to testers: there are DEPENDs against test use, don't forget to install them if you use ebuild directly.

### CODE REVIEW

File: app-text/po4a/po4a-0.47.ebuild [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+PLOCALES="af ca cs da de eo es et eu fr hr id it ja kn ko nb nl pl pt_BR pt ru sl sv uk vi zh_CN zh_HK"
+PLOCALES_BACKUP="en"
+
+# Needed because this package also installs to vendor_perl
+GENTOO_DEPEND_ON_PERL=noslotop
+inherit perl-module l10n
+
+DESCRIPTION="Tools for helping translation of documentation"
+HOMEPAGE="http://po4a.alioth.debian.org"
+SRC_URI="mirror://debian/pool/main/p/po4a/${PN}_${PV}.orig.tar.gz"
```
Comment (gktrk) [inline app-text/po4a/po4a-0.47.ebuild]: Can we switch to the arrow syntax here? i.e. `-> ${P}.tar.gz`

File: app-text/po4a/po4a-0.47.ebuild [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		app-text/docbook-sgml-utils
+		virtual/tex-base )"
+
+# Running tests in parallel fails
+DIST_TEST="do"
+
+src_prepare() {
+	# Check against locale files in ${S}/pod/bin for mismatches
+	# with languages listed in PLOCALES
+	local locales_path="$S/po/bin"
```
Comment (gktrk) [inline app-text/po4a/po4a-0.47.ebuild]: `${S}`

File: app-text/po4a/po4a-0.47.ebuild [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	l10n_find_plocales_changes "$locales_path" "" ".po"
```
Comment (gktrk) [inline app-text/po4a/po4a-0.47.ebuild]: `${locales_path}`

File: app-text/po4a/po4a-0.47.ebuild [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+	# Get rid of disabled locales
+	my_get_disabled_locales() {
+		PERL_RM_FILES=( "${PERL_RM_FILES[@]}" "po/bin/${1}.po" "po/pod/${1}.po" )
+	}
+
+	l10n_for_each_disabled_locale_do my_get_disabled_locales
+
+	einfo "Your LINGUAS lists the following languages: $LINGUAS"
```
Comment (gktrk) [inline app-text/po4a/po4a-0.47.ebuild]: `${LINGUAS}`

File: app-text/po4a/po4a-0.47.ebuild (hunk 1 above)
Comment (Fat-Zer) [inline app-text/po4a/po4a-0.47.ebuild]: yes, but why should we?

File: app-text/po4a/po4a-0.47.ebuild (hunk 2 above)
Comment (Fat-Zer) [inline app-text/po4a/po4a-0.47.ebuild]: is this syntax required by upcoming EAPI? or some styleguide? or whatever?

Comment (gktrk) [inline app-text/po4a/po4a-0.47.ebuild]: It is to maintain consistency across the ebuild

File: app-text/po4a/po4a-0.47.ebuild (hunk 1 above)
Comment (gktrk) [inline app-text/po4a/po4a-0.47.ebuild]: `.orig` doesn't do much there and it unpacks to ${P} anyway

Comment (Fat-Zer) [inline app-text/po4a/po4a-0.47.ebuild]: I don't really like to do so... "orig.tar.gz" is used all over the tree for mirror://debian. And only few ebuilds rename it... And in 3 of 4 cases such rename is caused by historical reasons (the ebuilds used to have a different SRC_URI with a "traditional" file name)...

File: app-text/po4a/po4a-0.47.ebuild (hunk 2 above)
Comment (Fat-Zer) [inline app-text/po4a/po4a-0.47.ebuild]: fixed

**Comment by gktrk** [comment]
Merged in https://github.com/gentoo/gentoo/commit/e64a786b623af3a08ee3f1a5bf8e654e4fdfa882. Thanks!

**Pull request closed without merging**